syntax = "proto3";

package Storage;

message Result {
	float light_intensity = 1;
}